.Rprofile

```diff
@@ -1,5 +1,5 @@
 .First <- function(){
-  if (require(devtools))
-    as.package(".")
+  if (requireNamespace("devtools", quietly = TRUE))
+    devtools::as.package(".")
```

Comment:
I'm not entirely sure what this accomplishes. Maybe `.Rprofile` could just be removed?